lake/Visualisations.py

```diff
@@ -0,0 +1,111 @@
+import os, re
+import matplotlib.pyplot as plt
+import seaborn as sns
+import torch
+import csv
+
+experiment = 'pairs_structure'
```

Comment:
Do you think you could change these to make the visualisation script parameterisable from the command line? Kind of similar to the arguments in oneshot_cls using `argparse`

Reply:
I created a class to plot the heat maps and now the class is called during the pairs experiment.